src/build/get_build_environment.mjs

```diff
@@ -23,9 +23,9 @@ import {getBuildNumber} from './get_build_number.mjs';
   Outputs:
   => the build environment object
 */
-export async function getBuildEnvironment(platform, buildMode) {
+export async function getBuildEnvironment(platform, buildMode, {sentryDsn} = {}) {
```

Comment:
You always call getBuildEnvironment with sentryDsn specified. Make the parameter mandatory.
```suggestion
export async function getBuildEnvironment(platform, buildMode, sentryDsn) {
```

Reply:
Okay, I see what you mean, done.